packages/vscode-extension/src/startup/DevAssistConfiguration.ts

```diff
@@ -176,15 +181,16 @@ const showDevAssistStartUpNotification = () => {
 
 const showStartDevAssistProblemNotification = (errorStage: string, error: string, devAssistStatusBar: vscode.StatusBarItem) => {
     // console.log(`There was a problem when starting DevAssist service. (${errorStage})\n${error}`)
-    setErrorDevAssistStausBarMessage(devAssistStatusBar)
+    setErrorDevAssistStausBarMessage(devAssistStatusBar);
+    vsLogger.printTimestamp();
```

Comment:
I'm not sure that it's a good idea to add the timestamp here, as this method is usually called as an already ongoing action that most probably has already shown the timestamp.

I'll need to check it a little bit further to be sure about it.